### PR TITLE
Feature/experiencefragment component

### PR DESCRIPTION
**Pull request by Rajamouli-hcl**
Added Experiencefragment component

### CODE REVIEW

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml [hunk 1]
```diff
@@ -40,6 +40,15 @@
                         components="[group:hclecomm.Content]">
                         <jcr:content jcr:primaryType="nt:unstructured"/>
                     </policy_600116016309972>
+		    <policy_414135893823493
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml]: @Rajamouli-hcl : where is this policy being used in which template ?

File: ui.content/src/main/content/jcr_root/content/experience-fragments/hclecomm/language-masters/en/site/simple-experience-fragment/master/.content.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rishabh-behl) [inline ui.content/src/main/content/jcr_root/content/experience-fragments/hclecomm/language-masters/en/site/simple-experience-fragment/master/.content.xml]: @Rajamouli-hcl : we dont need to checkin the exp fragment content as part of codebase

**Review by rishabh-behl — COMMENTED** [review]
@Rajamouli-hcl : Pl check comments

File: ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml (hunk 1 above)
Comment (Rajamouli-hcl) [reply ui.content/src/main/content/jcr_root/conf/hclecomm/settings/wcm/policies/.content.xml]: reused content-ploicy in experience-fragment web variation template and  removed old policy which i have created previously

File: ui.content/src/main/content/jcr_root/content/experience-fragments/hclecomm/language-masters/en/site/simple-experience-fragment/master/.content.xml (hunk 2 above)
Comment (Rajamouli-hcl) [reply ui.content/src/main/content/jcr_root/content/experience-fragments/hclecomm/language-masters/en/site/simple-experience-fragment/master/.content.xml]: changes updated as per your review

**Review by rishabh-behl — COMMENTED** [review]
LGTM

**Pull request merged**